tutorials/review.adoc

```diff
@@ -0,0 +1,881 @@
+* IMPORTANT
+==== <4>
+------
+
+If an admonition is valid for a whole section or a whole document and should appear before any content in the section, you can set the `beforeclauses` attribute like this:
+
+[source, AsciiDoc]
+------
+[IMPORTANT,beforeclauses=true]
+====
+This important notice applies to the entire document.
+====
+------
+
+[[exercise-2-7]]
+include::exercises.adoc[tag=exercise-2-7]
+
+Great progress so far! Let's look at code samples in the next lesson.
+>> Start next lesson
+
+==== Code Samples
+
+AsciiDoc supports code samples using the `[source]` attribute before a new block. If you specify the coding language, the sample will be highlighted correctly. Have a look at the example:
+
+[source, AsciiDoc]
+------
+.Sample Code <1>
+[source,ruby] <2>
+==== <3>
+puts "Hello, world." <4>
+%w{a b c}.each do |x|
+  puts x
+end 
+==== <5>
+------
+
+<1> Title for your code sample (optional)
+<2> `[source]` specifies that the block contains code. `ruby` is the coding language. The block will highlight keywords depending on the coding language.
+<3> Starting delimiter
+<4> Sample code
+<5> Ending delimiter
+
+[[exercise-2-8]]
+include::exercises.adoc[tag=exercise-2-8]
+
+We’re done with blocks - good job! The next lesson covers inline markup. 
+>> Start next lesson
+
+=== Inline Markup
+
+To annotate single words, you use inline markup. The markup encloses the word(s) and you do not need to begin a new line. Inline markup allows you to:
+
+* Emphasize text
+* Link to external resources with hyperlinks
+* Create internal cross-references
+* Create index entries
+* Create Bibliographic entries
+
+Let’s look at these features in the following lessons.
+>> Start next lesson
+
+==== Text markup
+
+The simplest form of inline markup is to emphasize text. 
+AsciiDoc allows you to:
+
+* Make words *bold* using asterisks 
+* _Italicise_ words with underscores 
+* Apply `monospacing` using backticks
+
+[source, AsciiDoc]
+---- 
+*bold*
+_italic_
+`monospace`
+----
+
+Metanorma extends these simple formats with:
+
+* Strikethrough text ([strike]#strike through text#)
+* Smallcaps text ([smallcap]#small caps text#)
+* Underline text ([underline]#underline text#)
+
+[source, AsciiDoc]
+---- 
+[strike]#strike through text#
+[smallcap]#small caps text#
+[underline]#underline text#
+----
+
+Metanorma also supports automatic character replacements, for example the copyright symbol © can be entered using `(C)`. For more information about automatic character replacements, see https://docs.asciidoctor.org/asciidoc/latest/subs/replacements/[the AsciiDoc documentation]. 
+
+[[exercise-2-9]]
+include::exercises.adoc[tag=exercise-2-9]
+
+Let’s look at a more advanced form of inline formatting: index entries
+>> Start next lesson
+
+==== Index Terms
+
+Before you define an index term, your document needs a section where the index terms can appear. To create an index, define a level 1 section (==) marked with the style index at the end of your document.
```

Comment:
Not all flavours support indexes: they will be processed in the XML, but stripped out by default in rendering. Indexing is currently supported in ISO, IETF, and BIPM. Advise readers to check whether their flavour does.